src/i18n/languages.js

```diff
@@ -10,4 +10,5 @@ module.exports = {
   pl: 'Polish',
   ru: 'Русский',
   ua: 'Українська',
+  'es-ES': 'Español - España',
```

Comment:
as spanish is no subset of spanish (😄) we don't need the following `ES` in the key. Please change the key from 
`'es-ES'` to `es` (also not single quotes necessary).

Going into detail would make a lot of sense when we are e.g separating US & British English, German-, Austrian- and Swiss German, ...

Reply:
Hello!
Just did that because it is usual to have Spanish from Spain and Spanish from Mexico. It is how most software does it. I can change it in a minute anyway

Reply:
You are right. I did a quick check where I was reminded about how fragmented Spanish actually is. In that context, your solution makes a lot of sense.

Thanks! :)

Reply:
One more thing to discuss, I would like to discuss how to keep language names consistent.
https://github.com/meetfranz/franz/pull/132#discussion_r147218680

What are your two cents on this?

Reply:
That's a difficult question... 🤔
I guess that people who use non-latin languages are the ones who are going to be more affected about this. 
In my opinion, I prefer to see them in english, as everyone would be able to read it. You can read something in a language you don't know (pronunciation facts apart) if it is written in latin alphabet, but you can't do it if its not. But that is just my opinion. Maybe people who speak those languages prefer to see them written the way they are used to. 
Both options will be good, anyway! 😄